plugins/node/opentelemetry-instrumentation-restify/src/constants.ts

```diff
@@ -24,4 +24,4 @@ export const RESTIFY_METHODS = [
   'patch',
 ];
 export const MODULE_NAME = 'restify';
-export const SUPPORTED_VERSIONS = ['>=4.0.0 <9'];
+export const SUPPORTED_VERSIONS = ['>=4.0.0 <12'];
```

Comment:
This should also be updated in the [instrumentation package README](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-restify#supported-versions)